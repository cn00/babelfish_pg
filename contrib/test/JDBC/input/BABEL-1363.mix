-- NOTE: this test should not run in parallel due to DB config change.
GO

-- tsql
-- VariableSetStmt doens't work in JDBC. use workaround
DECLARE @orig_debug_parallel_query varchar(10);
SET @orig_debug_parallel_query = (SELECT current_setting('debug_parallel_query'));

SELECT 'enable debug_parallel_query' FROM (SELECT
		set_config('debug_parallel_query', 'on', false)) sq;
GO

-- tsql
create table babel_1363_t1 (a int);
insert into babel_1363_t1 values (1), (2), (3), (4), (5), (6);
GO

-- psql     currentSchema=master_dbo,public
explain (costs off) select * from babel_1363_t1;
GO

-- tsql
select * from babel_1363_t1;
GO

-- tsql
DECLARE @orig_debug_parallel_query varchar(10);
SET @orig_debug_parallel_query = (SELECT current_setting('debug_parallel_query'));
SELECT 'reset debug_parallel_query' FROM (SELECT
		set_config('debug_parallel_query', @orig_debug_parallel_query, false)) sq;
GO

-- tsql
DROP TABLE babel_1363_t1
GO
