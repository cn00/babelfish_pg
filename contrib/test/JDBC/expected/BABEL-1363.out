-- NOTE: this test should not run in parallel due to DB config change.
GO

-- tsql

-- VariableSetStmt doens't work in JDBC. use workaround
DECLARE @orig_debug_parallel_query varchar(10);
SET @orig_debug_parallel_query = (SELECT current_setting('debug_parallel_query'));
SELECT 'enable debug_parallel_query' FROM (SELECT
		set_config('debug_parallel_query', 'on', false)) sq;
GO
~~START~~
varchar
enable debug_parallel_query
~~END~~


-- tsql
create table babel_1363_t1 (a int);
insert into babel_1363_t1 values (1), (2), (3), (4), (5), (6);
GO
~~ROW COUNT: 6~~


-- psql     currentSchema=master_dbo,public
explain (costs off) select * from babel_1363_t1;
GO
~~START~~
text
Seq Scan on babel_1363_t1
~~END~~


-- tsql
select * from babel_1363_t1;
GO
~~START~~
int
1
2
3
4
5
6
~~END~~


-- tsql
DECLARE @orig_debug_parallel_query varchar(10);
SET @orig_debug_parallel_query = (SELECT current_setting('debug_parallel_query'));
SELECT 'reset debug_parallel_query' FROM (SELECT
		set_config('debug_parallel_query', @orig_debug_parallel_query, false)) sq;
GO
~~START~~
varchar
reset debug_parallel_query
~~END~~


-- tsql
DROP TABLE babel_1363_t1
GO
